selftests/functional/test_multiplex.py

```diff
@@ -54,9 +54,10 @@ def test_mplex_plugin(self):
 
     def test_mplex_plugin_nonexistent(self):
         cmd_line = './scripts/avocado multiplex -m nonexist'
-        expected_rc = exit_codes.AVOCADO_JOB_FAIL
+        expected_rc = exit_codes.AVOCADO_ALL_OK
```

Comment:
Again, I see no reason to be so forgiving about such a failure.  Can you give some reasons for changing that behavior?